api/v1beta1/kafkacluster_types.go

```diff
@@ -36,6 +36,7 @@ type KafkaClusterSpec struct {
 	Brokers                []Broker                `json:"brokers"`
 	RollingUpgradeConfig   RollingUpgradeConfig    `json:"rollingUpgradeConfig"`
 	OneBrokerPerNode       bool                    `json:"oneBrokerPerNode"`
+	PropagateLabels        bool                    `json:"propagateLabels"`
```

Comment:
I think we should add an `omitempty` tag here. So it defaults to `false` without bothering the user.  Then the next `make manifests` will remove it from the required fields.